app/models/wallet.py

```diff
@@ -1,5 +1,37 @@
+from typing import List, Optional
+
+from aries_cloudcontroller.models.indy_cred_info import (
+    IndyCredInfo as IndyCredInfoAcaPy,
+)
+from aries_cloudcontroller.models.vc_record import VCRecord as VCRecordAcaPy
 from pydantic import BaseModel, Field
 
 
 class SetDidEndpointRequest(BaseModel):
     endpoint: str = Field(...)
+
+
+class VCRecord(VCRecordAcaPy):
+    credential_id: str = Field(
+        ..., alias="record_id", description="Rename record_id to credential_id"
+    )
+    record_id: str = Field(
+        ..., alias="credential_id", description="For backwards compatibility"
+    )
+
+
+class VCRecordList(BaseModel):
+    results: Optional[List[VCRecord]] = None
+
+
+class IndyCredInfo(IndyCredInfoAcaPy):
+    credential_id: str = Field(
+        ..., alias="referent", description="Rename referent to credential_id"
+    )
+    referent: str = Field(
+        ..., alias="credential_id", description="For backwards compatibility"
+    )
```

Comment:
I think the field descriptions here should be something else.

We're overriding the ACA-Py models, so for our new name (`credential_id`), let's use the ACA-Py description of record_id/referent (the one we're renaming), and then we keep the same description for record_id/referent (which we are preserving for backward compatibility), and just add in brackets (deprecated - renamed to credential_id). Something like that. Also see if Field() accepts a deprecated=True for those